src/App.tsx

```diff
@@ -77,7 +77,7 @@ export const App: React.FC = () => {
                 },
             }}
             variant={isMobile ? 'temporary' : 'permanent'}
-            activeItem={selected}
+            activeItem={selected !== '' ? selected : undefined}
```

Comment:
I think you can probably undo this now, since none of the routes will have `''` as their id anymore.